### DISCUSSION TITLE
The ping command is not working .

### DISCUSSION THREAD

**Discussion by Pipitapi** [arendst/Tasmota#18322] [Q&A]
Hello! 

I would like to ask for help! I can't figure out why the ping command is not working .
Rule1 ON Time#Minute|5 DO Ping4 192.168.0.2 ENDON ON Ping#192.168.0.2#Success==0 DO Backlog Power1 0; Delay 30; Power1 1; ENDON
I want to set this up, but the ping is not working. Thanks for your help.

Thanks for your help.

00:00:00.001 HDW: ESP8266EX
00:00:00.046 CFG: Flash-ről betöltve a(z) FB, Szám 27
00:00:00.176 QPC: Reset
00:00:00.204 Projekt tasmota - Tasmota Verzió 12.4.0(tasmota)-2_7_4_9(2023.02.24 15:38:38)
00:00:00.844 WIF: AP-hoz csatlakozás:1 Zsolt Channel 4 BSSId 04:D9:F5:C8:92:40 mód: 11n mint tasmota-7C19B4-6580...
00:00:01.752 WIF: Csatlakoztatva
00:00:02.004 HTP: Webszerver aktív: tasmota-7C19B4-6580 IP cím: 192.168.0.129
21:40:17.052 MQT: Csatlakozás...
21:40:17.061 MQT: Csatlakoztatva
21:40:17.064 MQT: tele/tasmota_7C19B4/LWT = Online (megtartott)
21:40:17.066 MQT: cmnd/tasmota_7C19B4/POWER = 
21:40:17.071 MQT: tele/tasmota_7C19B4/INFO1 = {"Info1":{"Module":"1 Channel","Version":"12.4.0(tasmota)","FallbackTopic":"cmnd/DVES_7C19B4_fb/","GroupTopic":"cmnd/tasmotas/"}}
21:40:17.087 MQT: tele/tasmota_7C19B4/INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-7C19B4-6580","IPAddress":"192.168.0.129"}}
21:40:17.099 MQT: tele/tasmota_7C19B4/INFO3 = {"Info3":{"RestartReason":"Software/System restart","BootCount":6}}
21:40:17.111 MQT: stat/tasmota_7C19B4/RESULT = {"POWER":"OFF"}
21:40:17.115 MQT: stat/tasmota_7C19B4/POWER = OFF
21:40:18.181 MQT: stat/tasmota_7C19B4/STATUS = {"Status":{"Module":12,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_7C19B4","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
21:40:18.229 MQT: stat/tasmota_7C19B4/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release-12.4.0/tasmota-HU.bin.gz","RestartReason":"Software/System restart","Uptime":"0T00:00:05","StartupUTC":"2023-04-02T20:40:13","Sleep":50,"CfgHolder":4617,"BootCount":5,"BCResetTime":"2023-04-02T21:20:51","SaveCount":28,"SaveAddress":"FB000"}}
21:40:18.282 MQT: stat/tasmota_7C19B4/STATUS11 = {"StatusSTS":{"Time":"2023-04-02T21:40:18","Uptime":"0T00:00:05","UptimeSec":5,"Heap":27,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":27,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":70,"Signal":-65,"LinkCount":1,"Downtime":"0T00:00:03"}}}
21:40:18.330 MQT: stat/tasmota_7C19B4/STATUS2 = {"StatusFWR":{"Version":"12.4.0(tasmota)","BuildDateTime":"2023.02.24 15:38:38","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"381/699"}}
21:40:18.382 MQT: stat/tasmota_7C19B4/RESULT = {"Time":"2023-04-02T21:40:18","Uptime":"0T00:00:05","UptimeSec":5,"Heap":27,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":31,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":70,"Signal":-65,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:40:21.403 MQT: tele/tasmota_7C19B4/STATE = {"Time":"2023-04-02T21:40:21","Uptime":"0T00:00:08","UptimeSec":8,"Heap":26,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":68,"Signal":-66,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:40:41.342 MQT: stat/tasmota_7C19B4/RESULT = {"POWER":"ON"}
21:40:41.347 MQT: stat/tasmota_7C19B4/POWER = ON
21:40:42.121 MQT: stat/tasmota_7C19B4/RESULT = {"POWER":"OFF"}
21:40:42.124 MQT: stat/tasmota_7C19B4/POWER = OFF
21:41:13.751 CMD: Ping4 192.168.0.2
21:41:13.758 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:41:50.835 CMD: rule1 1
21:41:50.842 MQT: stat/tasmota_7C19B4/RESULT = {"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":121,"Free":390,"Rules":"ON Time#Minute|5 DO Ping4 192.168.0.2 ENDON ON Ping#192.168.0.2#Success==0 DO Backlog Power1 0; Delay 30; Power1 1; ENDON"}}
21:43:28.880 CMD: Ping4 192.168.0.2
21:43:28.888 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:43:36.757 CMD: Ping 192.168.0.2
21:43:36.766 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:43:47.526 CMD: Ping0 192.168.0.2
21:43:47.534 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:45:00.399 RUL: TIME#MINUTE|5 performs "Ping4 192.168.0.2"
21:45:00.407 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:45:21.401 MQT: tele/tasmota_7C19B4/STATE = {"Time":"2023-04-02T21:45:21","Uptime":"0T00:05:08","UptimeSec":308,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":72,"Signal":-64,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:50:00.419 RUL: TIME#MINUTE|5 performs "Ping4 192.168.0.2"
21:50:00.428 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:50:12.410 APP: Soros naplózás kikapcsolva
21:50:21.386 MQT: tele/tasmota_7C19B4/STATE = {"Time":"2023-04-02T21:50:21","Uptime":"0T00:10:08","UptimeSec":608,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":82,"Signal":-59,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:55:00.389 RUL: TIME#MINUTE|5 performs "Ping4 192.168.0.2"
21:55:00.396 MQT: stat/tasmota_7C19B4/RESULT = {"Command":"Unknown"}
21:55:21.417 MQT: tele/tasmota_7C19B4/STATE = {"Time":"2023-04-02T21:55:21","Uptime":"0T00:15:08","UptimeSec":908,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Wifi":{"AP":1,"SSId":"Zsolt","BSSId":"04:D9:F5:C8:92:40","Channel":4,"Mode":"11n","RSSI":82,"Signal":-59,"LinkCount":1,"Downtime":"0T00:00:03"}}

**Comment by barbudor** [comment]
Ping is not included by default and need self compile with `#define USE_PING`
